src/code/z_sub_s.c

```diff
@@ -1186,13 +1186,23 @@ Actor* SubS_FindActor(PlayState* play, Actor* actorListStart, u8 actorCategory,
     return actor;
 }
 
-s32 SubS_FillLimbRotTables(PlayState* play, s16* limbRotTableY, s16* limbRotTableZ, s32 numLimbs) {
-    s32 i;
+/**
+ * Fills two tables with rotation angles that can be used to simulate idle animations.
+ *
+ * The rotation angels are dependent on the current frame, so should be updated regularly, generally every frame.
+ *
+ * This is done for the desired limb by taking either the `sin` of the yTable value or the `cos` of th zTable value,
+ * multiplying by some scale factor (generally 200.0f), and adding that to the already existing rotation.
+ *
+ * Note: This effect is practically unnoticeable if the current animation already has motion involved.
```

Comment:
Would the unnoticeable affect be more of a result of how it's used by the actor and not as a consequence of the table itself? It's seems fairly consistent to use it as:
```c
            rot->y += (s16)(Math_SinS(this->fidgetTableY[fidgetIndex]) * 200.0f);
            rot->z += (s16)(Math_CosS(this->fidgetTableZ[fidgetIndex]) * 200.0f);
```
But you wouldn't have to? Or would the effect not work in other ways?

Reply:
Or I guess you do mention it above so maybe it's fine

Reply:
Nah you are right, changing the scale to be higher would let it be more noticeable. Added a comment to clarify that if the common scale of 200 is used, than it is not really noticeable.